pyneuroml/plot/Plot.py

```diff
@@ -240,6 +246,21 @@ def generate_plot(
         logger.info("Saved image to %s of plot: %s" % (save_figure_to, title))
 
     if show_plot_already:
+        if animate:
+            duration = 5
+            num_frames = len(xvalues[0])
+            interval = duration * 1000 / num_frames
```

Comment:
yes, I will update this. 
Interval is simply " Delay between frames in milliseconds", it can be harcoded. Most tutorial I have seen uses 50 or 100 ms.

Reply:
Cool--yeh, I think 50ms (20fps) is good for a default minimum value.